templates/layouts/_base.html

```diff
@@ -49,11 +49,15 @@
 
     <!-- Google Tag Manager -->
     <script nonce="{{ csp_nonce() }}">
+      // depending on the cookie settings this if will allow the gtm snippet to load or stop it from loading
+      var gtmCookieSettings = /^(.*)?\s*'usage':true\s*[^;]+(.*)?$/;
+      if (document.cookie.match(gtmCookieSettings)) {
```

Comment:
We shouldn't need to do this. The cookie could be read by flask and the `google_tag_manager_*` template variables not set if not enabled.